docs/data/material/getting-started/templates/checkout/Review.tsx

```diff
@@ -1,87 +1,79 @@
-  { name: 'Card number', detail: 'xxxx-xxxx-xxxx-1234' },
-  { name: 'Expiry date', detail: '04/2024' },
+  { name: 'Card type:', detail: 'Visa' },
+  { name: 'Card holder:', detail: 'Mr John Smith' },
+  { name: 'Card number:', detail: 'xxxx-xxxx-xxxx-1234' },
+  { name: 'Expiry date:', detail: '04/2024' },
 ];
 
 export default function Review() {
   return (
-    <React.Fragment>
-      <Typography variant="h6" gutterBottom>
-        Order summary
-      </Typography>
+    <Stack spacing={2}>
       <List disablePadding>
-        {products.map((product) => (
-          <ListItem key={product.name} sx={{ py: 1, px: 0 }}>
-            <ListItemText primary={product.name} secondary={product.desc} />
-            <Typography variant="body2">{product.price}</Typography>
-          </ListItem>
-        ))}
+        <ListItem sx={{ py: 1, px: 0 }}>
+          <ListItemText primary="Products" secondary="4 selected" />
+          <Typography variant="body2">$134.98</Typography>
+        </ListItem>
+        <ListItem sx={{ py: 1, px: 0 }}>
+          <ListItemText primary="Shipping" secondary="Plus taxes" />
+          <Typography variant="body2">$9.99</Typography>
+        </ListItem>
         <ListItem sx={{ py: 1, px: 0 }}>
           <ListItemText primary="Total" />
           <Typography variant="subtitle1" sx={{ fontWeight: 700 }}>
-            $34.06
+            $144.97
           </Typography>
         </ListItem>
       </List>
-      <Grid container spacing={2}>
-        <Grid item xs={12} sm={6}>
-          <Typography variant="h6" gutterBottom sx={{ mt: 2 }}>
-            Shipping
+      <Divider />
+      <Stack
+        direction="column"
+        divider={<Divider flexItem />}
+        spacing={2}
+        sx={{ my: 2 }}
+      >
+        <div>
+          <Typography variant="subtitle2" gutterBottom>
+            Shipment details
           </Typography>
           <Typography gutterBottom>John Smith</Typography>
-          <Typography gutterBottom>{addresses.join(', ')}</Typography>
-        </Grid>
-        <Grid item container direction="column" xs={12} sm={6}>
-          <Typography variant="h6" gutterBottom sx={{ mt: 2 }}>
+          <Typography color="text.secondary" gutterBottom>
+            {addresses.join(', ')}
+          </Typography>
+        </div>
+        <div>
+          <Typography variant="subtitle2" gutterBottom>
             Payment details
           </Typography>
           <Grid container>
             {payments.map((payment) => (
               <React.Fragment key={payment.name}>
-                <Grid item xs={6}>
-                  <Typography gutterBottom>{payment.name}</Typography>
-                </Grid>
-                <Grid item xs={6}>
-                  <Typography gutterBottom>{payment.detail}</Typography>
-                </Grid>
+                <Stack
+                  direction="row"
+                  alignItems="center"
+                  spacing={1}
```

Comment:
```suggestion
                  direction="row"
                  spacing={1}
                  useFlexGap
```

No need for the align-items there!